internal/services/controller/controller.go

```diff
@@ -432,33 +454,43 @@ func (c *Controller) debugQueueContent(maxItems int) string {
 	return content
 }
 
-func (c *Controller) getMissingConditionalInformers() []conditionalInformer {
-	var missingInformers []conditionalInformer
-	for _, informer := range c.conditionalInformers {
-		if _, ok := c.informers[informer.apiType]; !ok {
-			missingInformers = append(missingInformers, informer)
+func (c *Controller) informerHasAccess(ctx context.Context, informer conditionalInformer) bool {
+	//Cuts the groupName from the groupVersion, example: "apps/v1" -> "apps"
+	groupName, _ := strings.CutSuffix(informer.groupVersion, "/v1")
```

Comment:
The version isn't necessarily `v1`, it just so happens that the current informers all have this. But if another informer that is, e.g., `v1beta1` was added, then it wouldn't work and code would have to be adjusted.

Reply:
okay